envoy/server/factory_context.h

```diff
@@ -235,34 +235,25 @@ class GenericFactoryContext {
  * NOTE: this interface is used in proprietary access loggers, please do not delete
  * without reaching to Envoy maintainers first.
  */
-class ListenerAccessLogFactoryContext : public virtual CommonFactoryContext {
+class FactoryContext : public virtual CommonFactoryContext {
```

Comment:
@kyessenov need to update the comment of this class?   it looks like a little mismatch?